### PR TITLE
Remove redundant code

### PR DESCRIPTION
**Pull request by varun508**
Hey please review I have made some changes

### CODE REVIEW

File: src/pages/Dashboard.jsx [hunk 1]
```diff
@@ -1,12 +1,8 @@
-import { useContext } from "react";
-import { AuthContext } from "../contexts/AuthContext";
 import React from "react";
 import EventCard from "../components/EventCard";
 import EventModal from "../components/EventModal";
 
 const Dashboard = () => {
-  const { currentUser } = useContext(AuthContext);
```
Comment (tarunfy) [inline src/pages/Dashboard.jsx:8]: why did you remove this?

Comment (varun508) [reply src/pages/Dashboard.jsx:8]: abc
